### PR TITLE
Allow for dynamic bootstrap compilation

### PR DESCRIPTION
**Pull request by bwarren2**
This makes Grunt automatically rebuild Bootstrap CSS with tweaked variables for custom styling.  

This is a gesture toward a feature PR; is it desirable in the base cookiecutter?   It feels a little kludgy to name everything in bootstrap, but that is the way I think it has to be done.

Potential necessary changes: adding some docs to the README with the option chosen, editing CONTRIBUTORS.

### CODE REVIEW

**Comment by webyneter** [comment]
@bwarren2, thank you for your time! I strongly suggest that you document the PR.

**Comment by bwarren2** [comment]
Edits made to the README describing intended setup and workflow.

File: .gitignore [hunk 1]
```diff
@@ -42,3 +42,4 @@ celerybeat-schedule.db
 .coverage
 .tox
 .cache
+*.sublime-*
```
Comment (webyneter) [inline .gitignore]: I suggest that you pull this off into a separate PR.

**Comment by bwarren2** [comment]
Awesome!  Should I remove the .gitignore edit and force-push over this
branch, or open a new PR?

File: .gitignore (hunk 1 above)
Comment (webyneter) [inline .gitignore]: @bwarren2, please, comment in here so it is easy to follow up :)
> Awesome! Should I remove the .gitignore edit and force-push over this branch, or open a new PR?
We'd be better off with a new PR.

Comment (webyneter) [inline .gitignore]: @bwarren2, and don't forget to revert `.gitignore` commit in here.

Comment (bwarren2) [inline .gitignore]: Many apologies, I was on my phone.  Thank you for being so patient with me!  I'll make a new PR and mention this one for history.

Comment (webyneter) [inline .gitignore]: @bwarren2, no problem :)

**Comment by webyneter** [comment]
@bwarren2, did you really mean to close this?.

**Comment by webyneter** [comment]
@bwarren2, I guess, you didn't, so I'm re-opening this one.

**Comment by bwarren2** [comment]
Aye, that was unintentional, thank you!

**Comment by pydanny** [comment]
This looks good to me.
